src/SFML/Main/SFMLActivity.cpp

```diff
@@ -183,13 +183,11 @@ void ANativeActivity_onCreate(ANativeActivity* activity, void* savedState, size_
     loadLibrary("sfml-window", lJNIEnv, ObjectActivityInfo);
     loadLibrary("sfml-graphics", lJNIEnv, ObjectActivityInfo);
     loadLibrary("sfml-audio", lJNIEnv, ObjectActivityInfo);
-    loadLibrary("sfml-network", lJNIEnv, ObjectActivityInfo);
```

Comment:
Why was that removed?

Reply:
Bleh, my bad. Forgot I had an open PR.

Reply:
Just create an extra branch for the next PR. :)